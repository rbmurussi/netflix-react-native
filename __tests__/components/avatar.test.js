import React from "react";
import renderer from "react-test-renderer";
import Avatar from "../../components/Avatar";

jest.mock("@expo/vector-icons", () => {
  return { MaterialIcons: "Icon" };
});

test("Criando component", () => {
  expect(renderer.create(<Avatar edit />)).toMatchSnapshot();
});
