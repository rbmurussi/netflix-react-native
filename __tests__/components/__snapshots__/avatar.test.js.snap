// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Criando component 1`] = `
<View
  accessible={true}
  focusable={false}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "column",
      "opacity": 1,
    }
  }
>
  <View>
    <Image
      style={
        Array [
          Object {
            "height": 60,
            "marginBottom": 10,
            "marginLeft": 10,
            "marginRight": 10,
            "marginTop": 10,
            "width": 60,
          },
        ]
      }
    />
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "height": 60,
            "justifyContent": "center",
            "marginBottom": 10,
            "marginLeft": 10,
            "marginRight": 10,
            "marginTop": 10,
            "position": "absolute",
            "width": 60,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "black",
              "height": 60,
              "justifyContent": "center",
              "marginBottom": 10,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "opacity": 0.8,
              "position": "absolute",
              "width": 60,
            },
          ]
        }
      />
      <Icon
        color="white"
        name="edit"
        size={24}
      />
    </View>
  </View>
  <Text
    style={
      Array [
        Object {
          "color": "white",
        },
      ]
    }
  />
</View>
`;
